lib/session_builder.rb

```diff
@@ -24,10 +24,10 @@ def build_session!
     data = build_local_start_and_end_time(data)
 
     begin
-      session = Session.create!(data)
+      session = Session.create!(data.reject { |k| [:drawable, :deleted].include?(k)})
```

Comment:
What is a :drawable session? I'm just curious, cause I've never seen that.

Reply:
With Rails 4 trying to create a model w/ an attribute that doesn't exist on the model fails. 
```
User.create(pizza: "pants") // BOOM
```
Those two are coming from the mobile app.

This should be handled in a better way (selecting what's right and not removing what's wrong). But hopefully somebody else will stumble on this and improve